codejail_service/apps/api/urls.py

```diff
@@ -4,11 +4,11 @@
 All API URLs should be versioned, so urlpatterns should only
 contain namespaces for the active versions of the API.
 """
-from django.urls import include, re_path
+from django.urls import include, path
 
-from codejail_service.apps.api.v1 import urls as v1_urls
+from codejail_service.apps.api.v0 import urls as v0_urls
 
 app_name = 'api'
 urlpatterns = [
-    re_path(r'^v1/', include(v1_urls)),
+    path('v0/', include(v0_urls)),
```

Comment:
I'd call this v1. To me, v0 is when you are trying to work out an API, but the API has already been worked out in the eduNEXT version of the service, right?

Reply:
It's actually hardcoded as v0 in edxapp right now.